src/reactions/Reaction.js

```diff
@@ -8,6 +8,9 @@ import Emoji from '../emoji/Emoji';
 import { getAuth } from '../account/accountSelectors';
 import { emojiReactionAdd, emojiReactionRemove } from '../api';
 
+const REACTION_HEIGHT = 30;
+const SPINNER_OFFSET = (Platform.OS === 'ios') ? REACTION_HEIGHT : REACTION_HEIGHT - 2;
```

Comment:
Can we do this through https://github.com/zulip/zulip-mobile/blob/master/src/styles/platform.android.js and https://github.com/zulip/zulip-mobile/blob/master/src/styles/platform.ios.js